src/i18n/en.json

```diff
@@ -122,6 +123,28 @@
       "message-text-default": "Please recheck the form before confirmation",
       "button-text-ok": "Confirm",
       "button-text-cancel": "Cancel"
+    },
+
+    "fee": {
```

Comment:
let's split it to `fee-types` and `fee-table`

Reply:
the we won't need the `-fee` part

Reply:
Resolved, splitted